sdk/src/main/assets/hcaptcha-form.html

```diff
@@ -58,6 +58,27 @@
         }
     };
     var bridgeConfig = JSON.parse(BridgeObject.getConfig());
+    var hCaptchaID = null;
+
+    function execute(bridgeConfig, renderConfig) {
+        var rqdata = bridgeConfig.rqdata;
+        if (rqdata) {
+            hcaptcha.setData(hCaptchaID, {
+                rqdata: rqdata
+            });
+        }
+
+        if (renderConfig.size === 'invisible') {
+            hcaptcha.execute(hCaptchaID);
+        } else {
+            BridgeObject.onLoaded();
+        }
+    }
+
+    function reset() {
```

Comment:
Function name might be misleading. `resetAndExecute()` should be more appropriate.

Reply:
Fixed

Reply:
Would be nice to avoid force pushes as it makes it harder to diff on what was actually changed since last review. Upon merge, it is fine to do "squash and ff-merge". @CAMOBAP

Reply:
@DSergiu totally makes sense, sorry I forgot that we follow squash and ff-merge